Kernel/init.cpp

```diff
@@ -123,14 +123,14 @@ VFS* vfs;
     RefPtr<FloppyDiskDevice> fd0;
     RefPtr<FloppyDiskDevice> fd1;
     if ((detect >> 4) & 0x4) {
-        fd0 = FloppyDiskDevice::create(FloppyDiskDevice::DriveType::Master);
+        fd0 = FloppyDiskDevice::create(FloppyDiskDevice::DriveType::Master, 89, 0);
```

Comment:
This is kinda bad API. `FloppyDiskDevice` knows which major/minor to use based on the `DriveType` you're giving it.
Let it figure it out internally, instead of forcing the user of `FloppyDiskDevice::create()` to pass in the correct major/minor.